models/Events/DeliveryExecutionResultsRecalculated.php

```diff
@@ -32,14 +32,14 @@ class DeliveryExecutionResultsRecalculated implements Event
     private ?float $totalScore;
     private ?float $totalMaxScore;
     private bool $isFullyGraded;
-    private int $timestamp;
+    private ?int $timestamp;
 
     public function __construct(
         DeliveryExecutionInterface $deliveryExecution,
         ?float $totalScore,
         ?float $totalMaxScore,
         bool $isFullyGraded,
-        int $timestamp
+        ?int $timestamp
```

Comment:
Is there any case for it to be null?

Reply:
Yes, getFinishTime() can return null 
![image](https://github.com/oat-sa/extension-tao-outcome/assets/118974926/eae2da1d-bdbd-45ab-8091-e7f1b9acd610)

Reply:
but we do not trigger this event if the execution not finished, by it is obvious from it's name DeliveryExecutionResultsRecalculated, no results if it's not finished

Reply:
Makes sense, I will remove nulls